data/typelibrary/utils-1.0.0/typelib/assembling/BOOLS_TO_LWORD.fct

```diff
@@ -0,0 +1,295 @@
+			<VarDeclaration Name="IN_21" Type="BOOL" Comment="BIT 21"/>
+			<VarDeclaration Name="IN_22" Type="BOOL" Comment="BIT 22"/>
+			<VarDeclaration Name="IN_23" Type="BOOL" Comment="BIT 23"/>
+			<VarDeclaration Name="IN_24" Type="BOOL" Comment="BIT 24"/>
+			<VarDeclaration Name="IN_25" Type="BOOL" Comment="BIT 25"/>
+			<VarDeclaration Name="IN_26" Type="BOOL" Comment="BIT 26"/>
+			<VarDeclaration Name="IN_27" Type="BOOL" Comment="BIT 27"/>
+			<VarDeclaration Name="IN_28" Type="BOOL" Comment="BIT 28"/>
+			<VarDeclaration Name="IN_29" Type="BOOL" Comment="BIT 29"/>
+			<VarDeclaration Name="IN_30" Type="BOOL" Comment="BIT 30"/>
+			<VarDeclaration Name="IN_31" Type="BOOL" Comment="BIT 31"/>
+			<VarDeclaration Name="IN_32" Type="BOOL" Comment="BIT 32"/>
+			<VarDeclaration Name="IN_33" Type="BOOL" Comment="BIT 33"/>
+			<VarDeclaration Name="IN_34" Type="BOOL" Comment="BIT 34"/>
+			<VarDeclaration Name="IN_35" Type="BOOL" Comment="BIT 35"/>
+			<VarDeclaration Name="IN_36" Type="BOOL" Comment="BIT 36"/>
+			<VarDeclaration Name="IN_37" Type="BOOL" Comment="BIT 37"/>
+			<VarDeclaration Name="IN_38" Type="BOOL" Comment="BIT 38"/>
+			<VarDeclaration Name="IN_39" Type="BOOL" Comment="BIT 39"/>
+			<VarDeclaration Name="IN_40" Type="BOOL" Comment="BIT 40"/>
+			<VarDeclaration Name="IN_41" Type="BOOL" Comment="BIT 41"/>
+			<VarDeclaration Name="IN_42" Type="BOOL" Comment="BIT 42"/>
+			<VarDeclaration Name="IN_43" Type="BOOL" Comment="BIT 43"/>
+			<VarDeclaration Name="IN_44" Type="BOOL" Comment="BIT 44"/>
+			<VarDeclaration Name="IN_45" Type="BOOL" Comment="BIT 45"/>
+			<VarDeclaration Name="IN_46" Type="BOOL" Comment="BIT 46"/>
+			<VarDeclaration Name="IN_47" Type="BOOL" Comment="BIT 47"/>
+			<VarDeclaration Name="IN_48" Type="BOOL" Comment="BIT 48"/>
+			<VarDeclaration Name="IN_49" Type="BOOL" Comment="BIT 49"/>
+			<VarDeclaration Name="IN_50" Type="BOOL" Comment="BIT 50"/>
+			<VarDeclaration Name="IN_51" Type="BOOL" Comment="BIT 51"/>
+			<VarDeclaration Name="IN_52" Type="BOOL" Comment="BIT 52"/>
+			<VarDeclaration Name="IN_53" Type="BOOL" Comment="BIT 53"/>
+			<VarDeclaration Name="IN_54" Type="BOOL" Comment="BIT 54"/>
+			<VarDeclaration Name="IN_55" Type="BOOL" Comment="BIT 55"/>
+			<VarDeclaration Name="IN_56" Type="BOOL" Comment="BIT 56"/>
+			<VarDeclaration Name="IN_57" Type="BOOL" Comment="BIT 57"/>
+			<VarDeclaration Name="IN_58" Type="BOOL" Comment="BIT 58"/>
+			<VarDeclaration Name="IN_59" Type="BOOL" Comment="BIT 59"/>
+			<VarDeclaration Name="IN_60" Type="BOOL" Comment="BIT 60"/>
+			<VarDeclaration Name="IN_61" Type="BOOL" Comment="BIT 61"/>
+			<VarDeclaration Name="IN_62" Type="BOOL" Comment="BIT 62"/>
+			<VarDeclaration Name="IN_63" Type="BOOL" Comment="BIT 63"/>
+		</InputVars>
+		<OutputVars>
+			<VarDeclaration Name="OUT" Type="LWORD" Comment="LWORD Output"/>
+		</OutputVars>
+	</InterfaceList>
+	<FunctionBody>
+		<ST><![CDATA[FUNCTION BOOLS_TO_LWORD
```

Comment:
Why does this function not utilize a return value like the other functions, but a VAR_OUTPUT?

Reply:
we will change it. RETURN is much better than VAR\_OUTPUT, because the Function is then usable in ST as well.